en/FAQgeneral.md

```diff
@@ -81,6 +81,10 @@ One entry should appear.
 Just push “OK” to insert the entry into the database.
 For further information, please, consult the dedicated help file about the [DOI-to-BibTeX fetcher](DOItoBibTeX).
 
+#### Q: I have a valid DOI that is found by the web browser but not by JabRef. Why can't JabRef find my DOI through web search?
+
+A: Your DOI is most likely not listed in the [CrossRef database](https://search.crossref.org/), which is the service that JabRef uses to search for DOIs.
```

Comment:
I think this is a bit confusing for the user. We query doi.org (which is provided by Crossref).
In addition, I had the case that some DOIs were actually found through the browser, but returned invalid bibtex data in JabRef or just html.
See for example: https://github.com/JabRef/jabref/issues/5149

Reply:
Should I just remove this addition then? If not, how should I update it to more accurately reflect the missing DOIs issue?

Reply:
Reformulate the question to "Why can't find JabRef find any DOI?` and then list the possible reasons for this.
e.g. DOI.org returns invalid bibtex or if you use the Crossref fetcher, it may simply ne not in the database.

Reply:
@Siedlerchr I just pushed a new commit with the question and answer reformulated per your suggestions. Let me know if further changes are needed.